app/components/pipeline/CreateBuildDialog.js

```diff
@@ -77,50 +75,29 @@ class CreateBuildDialog extends React.PureComponent {
               required={true}
             />
 
-            {this.renderMoreOptions()}
+            <CollapsableArea label="Options" maxHeight={250}>
+              <FormTextarea
+                name="build[env]"
+                label="Environment Variables"
+                help="Place each environment variable on a new line, in the format <code>KEY=value</code>"
+                rows={3}
+              />
+              <div style={{ paddingLeft: 18 }}>
+                <input type="hidden" name="build[clean_checkout]" value="0" />
+                <label className="bold"><input name="build[clean_checkout]" type="checkbox" className="absolute" style={{ marginLeft: -18 }} value="1" />Clean checkout</label>
```

Comment:
This checkbox’s selection outline gets occluded by the outer padding.
<img width="377" alt="screen shot 2017-04-11 at 3 11 27 pm" src="https://cloud.githubusercontent.com/assets/282113/24893777/3971d0fe-1ec9-11e7-8146-a97cf316ce1a.png">